### ISSUE TITLE
Could we publish Hysteria Plugin on F-Droid?

### ISSUE THREAD

**Issue by linsui** [SagerNet/SagerNet#359]
I'd like to publish this plugin on F-Droid and make it updated automic as the main app. Is that OK? Thanks!

The fdroid script has a problem that it use go 1.18beta1 which failed building the plugin.

**Comment by nekohasekai** [comment]
Of course. For plugins, maybe need to modify prebuild for each plugin.

**Comment by linsui** [comment]
Thanks! I'll patch it before it's fixed.

**Comment by Molyuu** [comment]
I have checked and it's already there!

**Comment by linsui** [comment]
Wow, two more plugins. Could I package them?

**Comment by Molyuu** [comment]
Sorry dude I am not a maintainer.Please ask someone else.

**Comment by linsui** [comment]
@nekohasekai Kindly ping.

**Comment by linsui** [comment]
@nekohasekai Ping :)

**Comment by nekohasekai** [comment]
Ok.

**Comment by linsui** [comment]
Thanks!
